rambo/app.py

```diff
@@ -402,6 +402,9 @@ def up(ctx=None, **params):
     elif params.get('destroy_on_error') is False:
         cmd = '{} {}'.format(cmd, '--no-destroy-on-error')
 
+    if params.get('sync_dir'):
```

Comment:
This code is fine, but for consistency I think it should be moved to options.py and be called like the other options in L377-L382. This is currently with the flags which directly modify the pass-though cmd.